src/Internal/Plutip/Types.purs

```diff
@@ -95,17 +108,37 @@ data InitialUTxOsWithStakeKey =
 
 type InitialUTxODistribution = Array InitialUTxOs
 
-newtype ClusterStartupRequest = ClusterStartupRequest
-  { keysToGenerate :: InitialUTxODistribution
-  , epochSize :: UInt
-  , slotLength :: Seconds
-  }
+data ClusterStartupRequest
```

Comment:
Similar to above, ideally I think we'd remove the constructors. I was trying to find if plutip has a `StartClusterRequestWithConfig` endpoint, but I could only find `StartClusterRequest`, so I'm not sure I'm fully understanding the `EncodeAeson` instance.

Reply:
Actually I just saw https://github.com/mlabs-haskell/plutip/pull/165, I was looking at the commit from the flake. With the design from that PR I can see now why we're doing it this way. [I suggested an alternative design there](https://github.com/mlabs-haskell/plutip/pull/165#discussion_r1051504704), let me know what you think.

Reply:
done in 38cdff7